packages/odf/hooks/useSafeK8sGet.ts

```diff
@@ -0,0 +1,30 @@
+import { useK8sGet } from '@odf/shared';
+import { getValidK8sOptions } from '@odf/shared/utils';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+import { K8sKind } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+import { useOprInstallNamespaceSelector } from '../redux';
+
+/**
+ * Wrapper around "useK8sGet" hook.
+ * Ensures no API request is made unless its "safe" to do so (ODF installed namespace is fetched successfully).
+ */
+export const useSafeK8sGet = <R extends K8sResourceCommon = K8sResourceCommon>(
+  kind: K8sKind,
+  name?: string,
+  namespace?: string,
+  cluster?: string,
+  allowFallback = false
+): [R, boolean, unknown] => {
+  const { oprInstallNs, isNsSafe, isFallbackSafe } =
+    useOprInstallNamespaceSelector();
+
+  const canUseFallback = allowFallback && isFallbackSafe;
+  const useK8sGetArgs = getValidK8sOptions(
```

Comment:
```suggestion
  const K8sGetArgs = getValidK8sOptions(
```
 `use` should be prefix for hooks.

Reply:
ack...